src/App.tsx

```diff
@@ -10,32 +12,28 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabsId, setTabsId] = useState('tab-1');
+  const handleClick = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    const targetTab = event.currentTarget as HTMLAnchorElement;
+
+    setTabsId(targetTab.hash.slice(1));
+  };
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabsId);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is {selectedTab?.title}
       </h1>
 
       <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
+        <Tabs
+          tabs={tabs}
+          selectedTabsId={selectedTabsId}
```

Comment:
The `Tabs` component should receive only necessary props. Remove the `selectedTab` prop, as it is not needed inside the `Tabs` component.